src/command_modules/azure-cli-batchai/HISTORY.rst

```diff
@@ -3,16 +3,37 @@
 Release History
 ===============
 
-0.1.6
-+++++
-* Support Autorest 3.0 based SDKs
-
-0.1.5
+1.0.0
```

Comment:
Is this intended to be a GA release? If so, you'll need to schedule a full command review with the CLI team. Also... for reasons... the GA version is always "2.0.0".

Reply:
Can we schedule the review asap, please?

Reply:
So as discussed we'll just call this release 0.1.6 or 0.2.0 while we work through a few more items that warrant further discussion within the CLI team.

Reply:
0.2.0